javaslang/src/test/java/javaslang/collection/AbstractSeqTest.java

```diff
@@ -1822,6 +1822,18 @@ public void shouldTransposeIfEmpty() {
     }
 
     @Test
+    public void shouldTransposeIfIs1x0() {
```

Comment:
From what I can see the other tests _logically_ serve a different purpose. They test if the contents are the same. The identity test ensure that the references are the same. Technically they overlap but the semantics are different. I would leave them for now.